nemo_aligner/utils/utils.py

```diff
@@ -90,7 +97,7 @@ def load_from_nemo(
         save_restore_connector=connector,
         strict=strict,
     )
-    return model
+    return model, model_cfg if return_updated_cfg else model
```

Comment:
if you want the model_cfg can you use this function instead?

[load_checkpoint_model_config](https://github.com/NVIDIA/NeMo-Aligner/blob/main/nemo_aligner/utils/utils.py#L96)

Reply:
What we need here is the model_cfg after applying `modify_config_fn` , I don't see how we can replace with this function

Reply:
ah sorry good point -- the SFT configs work a bit differently than the other trainers(a long standing TODO to fix). this shouldn't block your PR so resolving it